test/parallel/test-runner-mocking.js

```diff
@@ -891,6 +891,55 @@ test('mock implementation can be changed dynamically', (t) => {
   assert.strictEqual(fn.mock.callCount(), 12);
 });
 
```

Comment:
Please include https://github.com/nodejs/node/pull/49088#discussion_r1289283473 as a test.

Reply:
I already did by updating the second test